site/metrics.md

```diff
@@ -20,6 +20,19 @@ exposed:
 * `weave_dns_entries` - Number of DNS entries.
 * `weave_flows` - Number of FastDP flows.
 
+#### Publish Router Metrics Endpoint
+
+By default, when started via `weave launch`, weave listens on its local 
+interface to serve metrics. To publish your metrics throughout your cluster, 
+e.g. if your prometheus server is installed on a different host machine,
+you need to set `WEAVE_STATUS_ADDR` to your corresponding IP and port.
+Default port is 6782.
+
+`WEAVE_STATUS_ADDR=X.X.X.X:<IP>`
```

Comment:
where you have 'IP' I think we'd call it 'port'; also we use angle brackets to denote something a user has to add within a bigger command so just leave those out.

So: 

    ```
    WEAVE_STATUS_ADDR=X.X.X.X:PORT
    ```